### PR TITLE
add urgent-care-tests and move docker to this directory

### PR DESCRIPTION
**Pull request by jtorano-va**
Related Story: https://vasdvp.atlassian.net/browse/CER-48

### CODE REVIEW

File: urgent-care-tests/pom.xml [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+            <groupId>io.fabric8</groupId>
+            <artifactId>docker-maven-plugin</artifactId>
+            <version>${docker-maven-plugin.version}</version>
+            <executions>
+              <execution>
+                <id>build-test</id>
+                <configuration>
+                  <images>
+                    <image>
+                      <name>${docker.organization}/health-apis-${project.artifactId}</name>
```
Comment (alecbert-va) [inline urgent-care-tests/pom.xml]: This is a reminder that we'll need an accompanying PR to update the name of the test artifact we use in the DU. Right now I think we just use "urgent-care-tests"

**Comment by alecbert-va** [comment]
This looks good but mock-ee follows the convention of having a `<relativePath/>` line within the parent block of the pom and I'm trying to remember what that does before merging.

**Comment by alecbert-va** [comment]
@dtournour-va does anything in the src folder of this project have a reason to live? I'd like to remove it as part of this PR after looking at it

**Comment by jtorano-va** [comment]
Tested additional commits locally, everything looks good.

**Pull request merged**